subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -230,6 +230,21 @@ This enables watching the file system for this build only.
 Put `org.gradle.unsafe.watch-fs=true` in your `gradle.properties`::
 This enables watching the file system for all builds, unless explicitly disabled with `--no-watch-fs`.
 
+Here are some operating system specific notes.
+
+===  Linux
+
+On Linux, file system watching uses inotify.
+Depending on the size of your build, it may be necessary to increase inotify limits.
```

Comment:
Should we somehow acknowledge the fact that if you are using an IDE already, then you probably already had to increase them?